pkgs/applications/editors/jetbrains/common.nix

```diff
@@ -28,7 +28,7 @@ with stdenv; lib.makeOverridable mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper patchelf p7zip unzip ];
 
-  patchPhase = ''
+  patchPhase = lib.optionalString (!stdenv.isDarwin) ''
```

Comment:
there is already `with stdenv.lib` above, line 7
all these `lib.` prefixes are non needed